### ISSUE TITLE
Transfer Learning on Mobilenet_SSD v2

### ISSUE THREAD

**Issue by kunalkumthekar** [dusty-nv/jetson-inference#1205]
I have already successfully performed transfer learning on mobilenet_ssd_v1  following  your tutorial. However, as we already have a link to download mb1_ssd.pth (pre-trained path file for mobilenet v1), can we get a link to download path file for mobilenet v2? I suppose after getting the pretrained mb_ssd_v2.pth file, the rest of the transfer learning process would be the same(i.e convert to onnx -> TRT engine -> execution through detectnet). Any help would be appreciated!

**Comment by dusty-nv** [comment]
@kunalkumthekar I haven't tried ssd-mobilenet-v2 with train_ssd.py and recall it not working with ONNX export.   There does not seem to be much advantage or increased accuracy to mobilenet-v2, either.   mobilenet-v2 enhancements are mostly aimed at mobile devices where separable convolutions have a bigger impact given the compute restraints.  However since Jetson has sufficient horsepower mobilenet-v1 is good.
